README.md

```diff
@@ -1,41 +1,47 @@
-![sxiv](http://muennich.github.com/sxiv/img/logo.png "sxiv")
+![nsxiv](https://raw.githubusercontent.com/nsxiv/nsxiv/gh-pages/img/logo.png "nsxiv")
 
-**Simple X Image Viewer**
+**Neo (or New or Not) Simple (or Small or Suckless) X Image Viewer**
```

Comment:
Out of curiosity, do you want to keep all these "alternative names" or to decide on one?

Reply:
https://github.com/nsxiv/nsxiv/discussions/47#discussioncomment-1333816